src/Functions/FunctionsConversion.h

```diff
@@ -634,11 +634,8 @@ struct ToDateTime64TransformFloat
         : scale(scale_)
     {}
 
-    inline NO_SANITIZE_UNDEFINED DateTime64::NativeType execute(FromType from, const DateLUTImpl &) const
+    NO_SANITIZE_UNDEFINED DateTime64::NativeType execute(FromType from, const DateLUTImpl &) const
     {
-        if (from < 0)
```

Comment:
Maybe some check like in line 621 would be great here

Reply:
Done, tnx.